src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
 
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+
+    public static void getStatistic(String fromFileName, String toFileName) throws IOException {
```

Comment:
method should not be static and throw IOException
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
```